eth/vm/opcode_values.py

```diff
@@ -56,6 +56,7 @@
 GASPRICE = 0x3a
 EXTCODESIZE = 0x3b
 EXTCODECOPY = 0x3c
+EXTCODEHASH = 0x3f
```

Comment:
*nitpick*

The other opcodes are in byte order, maybe put this after `RETURNDATACOPY`?

Reply:
I was thinking back and forth for a minute when I wrote it but then settled with alphabetical order as that is the chosen order at most other places (e.g. in mnemonics). But now that I think about it again, would probably have made sense to use byte order here to stay consistent within this location.